### PR TITLE
Add description to issuingAuthority property

### PR DESCRIPTION
**Pull request by goneall**
Fixes one of the TODO's.

### CODE REVIEW

**Review by kestewart — COMMENTED** [review]
Consider adding University as an entity?

File: model/Core/Properties/issuingAuthority.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+An entity that is authorized to issue identification credentials.
 
 ## Description
 
-A issuingAuthority is TODO
+The entity may be a government, non-profit or commercial enterprise.  The URI provides a unique identifier for the issuing authority.
```
[the remark this answers is not in the document]
Comment (goneall) [reply model/Core/Properties/issuingAuthority.md]: Updated to add `educational institution`.

@kestewart - let me know if that works.

[the remark this answers is not in the document]
Comment (kestewart) [reply model/Core/Properties/issuingAuthority.md]: That works.

[the remark this answers is not in the document]
Comment (goneall) [reply model/Core/Properties/issuingAuthority.md]: Could you give it a positive review - one more and I can merge :)

File: model/Core/Properties/issuingAuthority.md [hunk 2]
```diff
@@ -4,15 +4,14 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+An entity that is authorized to issue identification credentials.
 
 ## Description
 
-A issuingAuthority is TODO
+The entity may be a government, non-profit, educational institution or commercial enterprise.  The URI provides a unique identifier for the issuing authority.
```
Comment (zvr) [inline model/Core/Properties/issuingAuthority.md:11]:
```suggestion
An issuingAuthority is an entity that is authorized to issue identification credentials.

The entity may be a government, non-profit, educational institution, or commercial enterprise.  The URI provides a unique identifier for the issuing authority.
```

**Review by zvr — CHANGES_REQUESTED** [review]
The Description is not always supposed to be read together with Summary, so let's repeat some basic stuff.

**Review by zvr — APPROVED** [review]

**Review by kestewart — APPROVED** [review]
Thanks for the additonal clarification.

**Pull request merged**